storage/factory.go

```diff
@@ -83,6 +83,14 @@ type ArchiveFactory interface {
 	CreateArchiveSpanWriter() (spanstore.Writer, error)
 }
 
+var (
+	// ErrMetricStorageNotConfigured can be returned by the MetricsFactory when the metric storage is not configured.
+	ErrMetricStorageNotConfigured = errors.New("Metric storage not configured")
+
+	// ErrMetricStorageNotSupported can be returned by the MetricsFactory when the metric storage is not supported by the backend.
+	ErrMetricStorageNotSupported = errors.New("Metric storage not supported")
```

Comment:
if you are adding these, something must be returning them.

Reply:
I don't think these are needed now.